app_dart/test/request_handlers/refresh_chromebot_status_test.dart

```diff
@@ -168,9 +174,42 @@ void main() {
       expect(task.buildNumberList, '1');
     });
 
+    test('save data to BigQuery when task finishes', () async {
+      final Commit commit = Commit(key: config.db.emptyKey.append(Commit, id: 'abc'), sha: 'abc');
+      final Task task = Task(key: commit.key.append(Task, id: 123), commitKey: commit.key, status: Task.statusNew);
+      config.tabledataResourceApi = tabledataResourceApi;
+      config.db.values[commit.key] = commit;
+      config.db.values[task.key] = task;
+
+      final Map<BranchLuciBuilder, Map<String, List<LuciTask>>> luciTasks =
+          Map<BranchLuciBuilder, Map<String, List<LuciTask>>>.fromIterable(
+              await LuciBuilder.getProdBuilders('flutter', config),
```

Comment:
Does this make http requests? That would make tests flaky.

Reply:
No. It is using `FakeConfig`, which will not make http requests.